and-cli-workspace.js

```diff
@@ -104,16 +104,14 @@ require("./command-runner").run(async () => {
 
     // Clone user forks of AndcultureCode repositories
     // -----------------------------------------------
-    if (program.usernames == null) {
-        shell.exit(0);
-    }
-
-    const usernames = program.usernames.split(",");
+    if (program.usernames != null) {
+        const usernames = program.usernames.split(",");
 
-    await js.asyncForEach(usernames, async (username) => {
-        await cloneByUser(username);
-        echo.newLine();
-    });
+        await js.asyncForEach(usernames, async (username) => {
```

Comment:
not familiar with `asyncForEach`; is this basically mapping to an array of promises and using `Promise.all(promises)` so they're all running in parallel?

Reply:
Its a custom method I added. It runs a list of async functions synchronously which is what I want to to provide real-time stream of output. Though not _as_ performant as if I parallelized it, felt that might be a bit toward micro-optimization at this point.

Reply:
oh I was thinking you just just use `Promise.all` to run them all at the same time, unless having them always print in the same order is what you're after.